### ISSUE TITLE
Descriptions for aliases

### ISSUE THREAD

**Issue by Vandesm14** [nushell/nushell#10107]
### Related problem

I have a bunch of git aliases, such as `gb -> git branch` that I sometimes forget the aliases for or what they do. I have recently learned that you can add descriptions to functions, which is great! However, there does not seem to be a way to add a description for aliases, which is unfortunate.

### Describe the solution you'd like

Make it possible to add descriptions to aliases.

Example:
```nu
# List all local branches
export alias gb = git branch
```

When using the Ctrl+I view (which I do not know the name for), it would show the alias as opposed to this error:
```
/home/shane| gb
gb    This alias wraps an unknown external command.
```

It should be:
```
/home/shane| gb
gb    List all local branches
```

### Describe alternatives you've considered

Instead of using aliases, I could create functions instead, which can have descriptions. However, I have about 100 or so aliases, so that would be a lot!

**Comment by fdncred** [comment]
I have this in my config.nu
```
# abbreviation for git
alias g = git
# abbreviation for git status
alias gst = git status
# List all local branches
alias gb = git branch
```
I type `gb<tab>` and get this
![image](https://github.com/nushell/nushell/assets/343840/2a2f3b4e-1cba-438f-b836-61222fed1c68)

I'm on the latest main branch v0.84.1

**Comment by kubouch** [comment]
0.84.0 should fix this. Does it still happen on the newest version?

**Comment by Vandesm14** [comment]
I've upgraded to 0.84.0 but am still facing the issue as described:
![image](https://github.com/nushell/nushell/assets/23623276/45d2ad27-6b4e-43e8-bc47-5dca81f29206)

**Comment by kubouch** [comment]
I can't reproduce it. Mine says, “List all local branches”. Can you produce a minimal reproducible example to make sure we're on the same page?

**Comment by amtoine** [comment]
i cannot reproduce this on latest `main` by adding
```nushell
# List all local branches
export alias gb = git branch
```
to the end of my `config.nu`

**Comment by Vandesm14** [comment]
I have removed everything from my `config.nu` file and replaced it with the `gb` alias. I have also reinstalled nushell via `cargo install nu`.

Here is what my config.nu looks like:
![image](https://github.com/nushell/nushell/assets/23623276/8e6d9839-3b4f-43a4-b9c7-bc95b1be08e6)

Here is the nushell version and ensuring that the `gb` alias exists:
![image](https://github.com/nushell/nushell/assets/23623276/33b37576-5afc-48b3-9b2c-0068fad68912)

And here is me using the Ctrl+I view to check the description of `gb`:
![image](https://github.com/nushell/nushell/assets/23623276/04639e1f-97a7-4726-9cf8-e4ba3316d4ec)

_Note: I have restarted my terminal after updating `config.nu` and reinstalling `nu` to ensure I am in an up-to-date shell env._

**Comment by Vandesm14** [comment]
Update: I built nushell directly from main and was able to get it to work fine. Is this feature not out yet to 0.84.0? Or maybe this is Linux shenanigans with installation on my end.

**Comment by amtoine** [comment]
> Update: I built nushell directly from main and was able to get it to work fine. Is this feature not out yet to 0.84.0? Or maybe this is Linux shenanigans with installation on my end.
>
might be a fix that was pushed after the release :+1: 

as we all cannot reproduce this with the latest revisions of Nushell, let's close this issue :relieved: 
if you happen to see that problem again @Vandesm14, feel free to reopen :wink:

**Comment by kubouch** [comment]
The fix should be a part of 0.84, it was merged before the release. But since it began to work, let's keep it closed and if need be, the issue can be reopened.
